### PR TITLE
Video DVC Cache

### PR DESCRIPTION
**Pull request by SoyGema**
Insert video DVC Cache. 
The format follows the getting started structure series . 
If there is another Markdown style format I should follow, let me know 

Thanks ! Have a nice day

### CODE REVIEW

File: content/docs/command-reference/push.md [hunk 1]
```diff
@@ -188,6 +188,8 @@ Finally, we used `dvc status` to double check that all data had been uploaded.
 
 ## Example: What happens in the cache?
 
+https://youtu.be/pqnxQIORp-4
```
Comment (skshetry) [inline content/docs/command-reference/push.md:191]: In the end, the video seems to cut off while trying to show `.dvc/cache` contents.

Comment (jorgeorpinel) [reply content/docs/command-reference/push.md:191]: At 2:45, yeah.

**Comment by jorgeorpinel** [comment]
The Pr is good, thanks @SoyGema ! But there may be an issue with the video?

p.s. you can see the review app here: https://dvc-org-soygema-doc-cac-jxsgk3.herokuapp.com/doc/command-reference/push#example-what-happens-in-the-cache

File: content/docs/command-reference/push.md (hunk 1 above)
Comment (SoyGema) [reply content/docs/command-reference/push.md:191]: Fixed and waiting for youtube link in order to change it
Thanks

**Review by jorgeorpinel — APPROVED** [review]
Change approved, thanks @SoyGema ! Please update the video link before merging 🙂

**Comment by SoyGema** [comment]
@jorgeorpinel video changed !

**Pull request merged**